openreview/venue/venue.py

```diff
@@ -1009,6 +1015,125 @@ def set_SAC_ethics_review_process(self, sac_ethics_flag_duedate=None):
+            res = iThenticate_client.create_submission(
+                owner=submission.content["authorids"]["value"][0],
+                title=submission.content["title"]["value"],
+                timestamp=datetime.datetime.fromtimestamp(submission.tcdate/1000, tz=datetime.timezone.utc).strftime('%Y-%m-%dT%H:%M:%SZ'),
+                owner_first_name=submission.content["authors"]["value"][0].split(
+                    " ", 1
+                )[0],
+                owner_last_name=submission.content["authors"]["value"][0].split(
+                    " ", 1
+                )[1],
+                owner_email=self.client.get_profile(submission.content["authorids"]["value"][0]).content["emailsConfirmed"][0],
+                group_id=f"{self.id}/-/Submission",
+                group_context={
+                    "id": self.id,
+                    "name": self.name,
+                    "owners": [
+                    # {
+                    #     "id": "d7cf2650-c1c7-11e8-b568-0800200c9a66",
+                    #     "family_name": "test_instructor_first_name",
+                    #     "given_name": "test_instructor_last_name",
+                    #     "email": "instructor_email@test.com"
+                    # },
+                    # {
+                    #     "id": "7a62f070-c265-11e8-b568-0800200c9a66",
+                    #     "family_name": "test_instructor_2_first_name",
+                    #     "given_name": "test_instrutor_2_last_name",
+                    #     "email": "intructor_2_email@test.com"
+                    # }
+                    ]
+                },
+                group_type="ASSIGNMENT",
+            )
+            iThenticate_submission_id = res["id"]
+            
+
+            # Create a new edge with head as the note ID and tail as the iThenticate submission ID
+            iThenticate_edge = openreview.api.Edge(
+                invitation=self.get_iThenticate_plagiarism_check_invitation_id(),
+                head=submission.id,
+                tail=iThenticate_submission_id,
+                label="Created",
+                weight=-1,
+            )
+
+            
+            iThenticate_edge = self.client.post_edge(iThenticate_edge)
+
+            # Get the PDF for the submission
+            submission_file_binary_data = self.client.get_attachment(
+                id=submission.id, field_name="pdf"
+            )
+
+            submission_file_object = io.BytesIO(submission_file_binary_data)
+
+
+            # update the edge with status file sent
+            iThenticate_edge.label = "File Sent"
+            iThenticate_edge = self.client.post_edge(iThenticate_edge)
+
+
+            # Upload PDF to iThenticate
+            res = iThenticate_client.upload_submission(
+                submission_id=iThenticate_submission_id,
+                file_data=submission_file_object,
+                file_name=submission.content['title']['value']
```

Comment:
@carlosmondra don't we need to format the title here? what is it too large or contains markdown/mathjax?